canonical-web-and-design/jaas.ai
# Bump global-nav to 2.0.7

## Done

Bump global-nav to 2.0.7

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Verify global nav stills works as expected
